test/builder_test.dart

```diff
@@ -160,7 +160,8 @@ class Customer { }
 
 const _testGenPartContent = r'''// GENERATED CODE - DO NOT MODIFY BY HAND
 
-part of test_lib;
+part of 'test_lib.dart';
```

Comment:
Yes. If we want to push this before the analyzer is fixed, we'll also want to generate the appropriate '// ignore: '. WDYT?